--- ../external/tbb/CMakeLists.txt	2024-03-13 00:34:43
+++ ../external/tbb/CMakeLists copy.txt	2024-03-13 00:34:39
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
 
 # Enable CMake policies
 
@@ -93,7 +93,7 @@
     set(TBB_ARCH 64)
 endif()
 
-option(TBB_TEST "Enable testing" ON)
+option(TBB_TEST "Enable testing" OFF)
 option(TBB_EXAMPLES "Enable examples" OFF)
 option(TBB_STRICT "Treat compiler warnings as errors" ON)
 option(TBB_WINDOWS_DRIVER "Build as Universal Windows Driver (UWD)" OFF)
@@ -233,7 +233,7 @@
     # Allow specifying external TBB to test with.
     # Do not add main targets and installation instructions in that case.
     message(STATUS "Using external TBB for testing")
-    find_package(TBB REQUIRED)
+    # find_package(TBB REQUIRED)
 else()
     if (TBB_BUILD)
         add_subdirectory(src/tbb)
@@ -273,9 +273,9 @@
             DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}
             COMPONENT devel)
 
-    install(FILES "README.md"
-            DESTINATION ${CMAKE_INSTALL_DOCDIR}
-            COMPONENT devel)
+    # install(FILES "README.md"
+    #         DESTINATION ${CMAKE_INSTALL_DOCDIR}
+    #         COMPONENT devel)
     # -------------------------------------------------------------------
 endif()
 
